migrations/versions/40d101e1211c_.py

```diff
@@ -0,0 +1,26 @@
+"""empty message
+
+Revision ID: 40d101e1211c
+Revises: 37e1e4549218
+Create Date: 2015-02-19 00:59:52.875733
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '40d101e1211c'
+down_revision = '37e1e4549218'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
```

Comment:
Note how these migrations did nothing. You need to delete the migration files your commits added, rm your app.db and rerun upgrade then migrate.